docs/context/ref/TransactionResult.md

```diff
@@ -0,0 +1,59 @@
+---
+id: TransactionResult
+sidebar_label: TransactionResult
+title: TransactionResult
+hide_title: true
+---
+# `TransactionResult`
+
+A context type representing the result of a transaction initiated via FDC3, which SHOULD be returned as an [`IntentResult`](../../api/Types#intentresult) by intents that create, retrieve, update or delete content or records in another application. Its purpose is to provide a status and message (where needed) for the transaction and MAY wrap a returned context object.
+
+## Type
+
+`fdc3.transactionResult`
+
+## Schema
+
+https://fdc3.finos.org/schemas/next/transactionresult.schema.json
+
+## Details
+
+| Property   | Type    | Required | Example Value     |
+|------------|---------|----------|-------------------|
+| `type`     | string  | Yes      | 'fdc3.transactionResult' |
+| `status`   | string  | Yes      | `"Created" | "Deleted" | "Updated" | "Failed"` |
```

Comment:
In the preview the Example Value shows as `"Created" , with none of the other values. I think the pipes are being parsed as column dividers

Reply:
fixed